cloudfoundry/provider/resource_app.go

```diff
@@ -96,6 +97,14 @@ func (r *appResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *
 				},
 				Optional: true,
 			},
+			"app_lifecycle": schema.StringAttribute{
```

Comment:
If the CNB type is included shouldn't there also be the option to specify the credentials for the different buildpacks (see API: https://v3-apidocs.cloudfoundry.org/version/3.200.0/index.html#lifecycles -> CNB)